pkgs/development/libraries/flite/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchFromGitHub, alsa-lib, fetchpatch }:
+{ lib, stdenv, fetchFromGitHub, alsa-lib, fetchpatch, libpulseaudio, withPulse ? false }:
```

Comment:
I think this should be more generic for longevity. I could see a PipeWire-based backend in the future. Or someone might need a package with OSS support as well in the future.

So maybe just make this a string like 
```
audioBackend ? "alsa"
```

And add the corresponding dependencies depending on the value.

Reply:
awesome idea! i just implemented this :)